src/reducers/index.js

```diff
@@ -91,6 +93,8 @@ const rootReducer = (state = initialState, action) => {
       return Object.assign({}, state, {addressesFound: action.payload.addressesFound, transactionsFound: action.payload.transactionsFound});
     case 'update_request_error_status_code':
       return Object.assign({}, state, {requestErrorStatusCode: action.payload});
+    case 'update_height':
+      return Object.assign({}, state, {height: action.payload.height});
```

Comment:
From what I've seen, this won't update the locked/unlocked balance automatically, right?

Reply:
For what we have here: https://react-redux.js.org/using-react-redux/connect-mapstate

> It is called every time the store state changes.

So every time the wallet receives an update on the height it will update the redux and call all `mapStateToProps` again, which updates the balance of each screen.

I will double check now only that all `calculateBalance` are called from `mapStateToProps`

Reply:
Ah ok, it checks again all state variables. Thanks